openshift/Dockerfile

```diff
@@ -0,0 +1,22 @@
+# This will produce and image to be used in OpenShift
+# Build should be triggered from the repo root like:
```

Comment:
I am not sure if building the image in OpenShift would be a good idea (might introduce some complications in terms of reproducibility). I would expect to have these images built by OSBS and deployed using this (or similar) template in a prod-like setup.

When building with OSBS the image will probably have it's own dist-git repo with a separate (but probably similar) Dockerfile. This Dockerfile mainly serves as an example, and maybe might be used by some CI setups.

Should there be a clarification somewhere to make clear that this is intended as an example?